packages/did-core-test-server/services/runSuite.js

```diff
@@ -8,7 +8,7 @@ module.exports = async (suiteName, config) => {
       // ci: true,
       // silent: true,
       roots: [path.resolve(__dirname, `../suites/${suiteName}`)],
-      globals: JSON.stringify({ suiteConfig: config }),
+      globals: JSON.stringify({ systemSuiteConfig: config }),
```

Comment:
for folks wondering, this is the magic of our jest tests... we are injecting a global json object here... which is the same object that an http client can submit to an http web server to generate the report json.

Reply:
Indeed. It was a shock to understand that `globals` is a jest's option. The name `globals` is obvious from the feature, but the name is strange for an optional parameter.